refactor-first-maven-plugin/pom.xml

```diff
@@ -31,6 +31,12 @@
             <version>3.6.1</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.13.2.2</version>
```

Comment:
If you could add this dependency to the parent pom and remove the version here, I would appreciate it.  This way we can update the version numbers in a single file instead of needing to go hunting for it.

Reply:
works for me, moved :-)